### PR TITLE
Use sources.list instead of sed to update sources.list

### PR DESCRIPTION
**Pull request by uilianries**
sed works, but it is complicated, which makes it difficult to understand what has been modified.

Changelog: Fix: Use sources.list instead of sed to update sources.list

- [ ] Refer to the issue that supports this Pull Request.
- [x] If the issue has missing info, explain the purpose/use case/pain/need that covers this Pull Request.
- [x] I've read the [Contributing guide](https://github.com/conan-io/conan-docker-tools/blob/master/.github/CONTRIBUTING.md).
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008) style guides for Python code.
- [x] I've followed the [Best Practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) guides for Dockerfile.

### CODE REVIEW

File: clang_3.9/sources.list [hunk 1]
```diff
@@ -0,0 +1,18 @@
+deb http://old-releases.ubuntu.com/ubuntu/ artful main restricted
```
Comment (SSE4) [inline clang_3.9/sources.list]: is this file different from `clang_3.9-x86/sources.list`?

Comment (uilianries) [reply clang_3.9/sources.list]: only a new line is different, but the content is the same. I checked with vimdiff

Comment (SSE4) [reply clang_3.9/sources.list]: okay, may we reduce copy-paste somehow? e.g. move file to some shared location, or just use it in base image once?

Comment (uilianries) [reply clang_3.9/sources.list]: I think the second options is better, since we can't use relative path when copying (docker feature).

Comment (uilianries) [reply clang_3.9/sources.list]: I cant use the second options ... docker-compose uses each image folder as a context, which means it won't the sources.list file. I could use symbolic link WDYT ?

Comment (SSE4) [reply clang_3.9/sources.list]: that's unfortunate, okay, let's leave it for now as is

**Pull request merged**